modules/installation-requirements-user-infra.adoc

```diff
@@ -111,7 +111,7 @@ endif::bare-metal,agnostic[]
 ifndef::openshift-origin[]
 Note that {op-system} is based on {op-system-base-full} 8 and inherits all of its hardware certifications and requirements.
 endif::[]
-See link:https://access.redhat.com/articles/rhel-limits[Red Hat Enterprise Linux technology capabilities and limits].
+See link:!https://access.redhat.com/articles/rhel-limits[Red Hat Enterprise Linux technology capabilities and limits].
```

Comment:
This is strange...I tried the link just now and it works for me.

Reply:
@bobfuru works today, odd.